lib/ansible/modules/net_tools/ldap/ldap_entry.py

```diff
@@ -204,6 +205,29 @@ def main():
         module.fail_json(msg=missing_required_lib('python-ldap'),
                          exception=LDAP_IMP_ERR)
 
+    # For Ansible-2.9.x and below, allow the params module parameter with a warning
+    if LooseVersion(module.ansible_version) < LooseVersion('2.10'):
+        if module.params['params']:
+            module.deprecate("The `params` option to ldap_attr will be removed in Ansible 2.10"
+                             " since it circumvents Ansible's option handling", version='2.10')
+
+        # However, the bind_pw parameter contains a password so it **must** go through the normal
```

Comment:
The code from here until line 224 needs to be indented by one level. Otherwise the module will crash if `params` is not used.

Reply:
Thanks for catching that!

Reply:
I think you have to do the same for ldap_attr. Anyway, with the commit you added ldap_entry no longer crashes for me (my playbooks no longer use ldap_attr, I vendored the 2.10 ldap_attrs a longer time ago).